documentation/plain_rules.md

```diff
@@ -1212,31 +1218,53 @@ Additionally all attributes are also included in non flatten format in the event
 
 ```json
 {
-    "noticeId": "799635b0-914f-11e6-836b-bf1691c99768",
-    "noticeTS": 1476368120971,
-    "id": "John Doe",
-    "type": "employee",
-    "isPattern": "false",
-    "subservice": "/",
-    "service": "unknownt",
-    "myJsonValue__color": "blue",
-    "myArrayValue__0": "green",
-    "myArrayValue__1": "black",
-    "stripped": [
-        {
-            "myJsonValue": {
-                "type": "myType1",
-                "value": {
-                    "color": "blue"
-                }
-            }
-        },
-        {
-            "myArrayValue": {
-                "type": "myType2",
-                "value": ["green", "black"]
-            }
-        }
-    ]
+	"noticeId": "7437b060-30c5-11ec-b47f-5f4885f1fa55",
+	"noticeTS": 1634638515814,
+	"id": "thing:disp6",
+	"type": "thing",
+	"isPattern": false,
+	"subservice": "/",
+	"service": "smartcity",
+	"TimeInstant__type": "DateTime",
+	"TimeInstant": "2021-10-19T10:15:37.050Z",
+	"TimeInstant__ts": 1634638537050,
+	"TimeInstant__day": 19,
+	"TimeInstant__month": 10,
+	"TimeInstant__year": 2021,
+	"TimeInstant__hour": 10,
+	"TimeInstant__minute": 15,
+	"TimeInstant__second": 37,
+	"TimeInstant__millisecond": 50,
+	"TimeInstant__dayUTC": 19,
+	"TimeInstant__monthUTC": 10,
+	"TimeInstant__yearUTC": 2021,
+	"TimeInstant__hourUTC": 10,
+	"TimeInstant__minuteUTC": 15,
+	"TimeInstant__secondUTC": 37,
+	"TimeInstant__millisecondUTC": 50,
+	"location__type": "Point",
+	"location__coordinates__0": 53.0859375,
+	"location__coordinates__1": 53.120405283,
+	"location__lat": 53.0859375,
+	"location__lon": 53.120405283,
+	"location__x": 642009.4673614734,
+	"location__y": 5883931.8311913265,
+	"stripped": {
+		"id": "thing:disp6",
+		"type": "thing",
+		"TimeInstant": {
+			"type": "DateTime",
+			"value": "2021-10-19T10:15:37.050Z",
+			"metadata": {}
+		},
+		"location": {
+			"type": "geo:json",
+			"value": {
+				"type": "Point",
+				"coordinates": [53.0859375, 53.120405283]
```

Comment:
As far as I remember, GeoJSON use long-lat ordering. Thus, this would not be consistent with

```
	"location__lat": 53.0859375,
	"location__lon": 53.120405283,
```

Is is just a typo in the documentation? Or the code is actually doing that way?

Reply:
related fix: https://github.com/telefonicaid/perseo-fe/pull/585